index.html

```diff
@@ -37,11 +37,11 @@ <h1 class="intro">I’m Simiran <br> Glad to see you!</h1>
                 <div class="body2">
```

Comment:
Great job @Simiran987 🤝

- [ ] Kindly make sure the text doesn't disappear or get cut off on the screen resize.
- [ ] Horizontal scrolling is not required to read sentences or "blocks of text". It is the best practice that when text size is increased, all the text in a sentence is visible.